lib/blacklight/models/wikipage.rb

```diff
@@ -8,6 +8,14 @@ def canvas_conversion(course, resources)
           select { |p| p.title.start_with? @title }.count
         @title = "#{@title}-#{page_count + 1}" if page_count > 0
         page = CanvasCc::CanvasCC::Models::Page.new
+        if !@url.empty?
+          @body = %{
```

Comment:
Would it make sense to append the url to the end instead of replacing the body? In case there was a description that also had a url?